tests/FieldTypesTest.php

```diff
@@ -74,8 +79,9 @@ public function testEmail4()
 
         $m->set('email_name', 'Romans <me@gmail.com>');
         $m->set('email_names', 'Romans1 <me1@gmail.com>, Romans2 <me2@gmail.com>; Romans3 <me3@gmail.com>');
-        $m->set('email_idn', 'test@日本レジストリサービス.jp');
+        $m->set('email_idn', 'test@háčkyčárky.cz'); // official IDN test domain
```

Comment:
not working with non-encoded IDN domain

Reply:
was a bug in your code, you checked with ascii domain only in format

Reply:
Romans Yellow Horse :) :) :)